webapp/channels/src/components/team_sidebar/components/team_button.tsx

```diff
@@ -36,186 +30,215 @@ interface Props {
+            );
+        } else if (isNotCreateTeamButton) {
+            teamClass = '';
+        } else {
+            teamClass = 'special';
+        }
+        ariaLabel = formatMessage({
+            id: 'team.button.unread.ariaLabel',
+            defaultMessage: '{teamName} team unread',
         },
         {
             teamName: displayName,
         });
 
-        if (!teamClass) {
-            if (unread && !this.props.isInProduct) {
-                teamClass = 'unread';
-
-                badge = (
-                    <span className={'unread-badge'}/>
-                );
-            } else if (isNotCreateTeamButton) {
-                teamClass = '';
-            } else {
-                teamClass = 'special';
-            }
+        if (mentions) {
             ariaLabel = formatMessage({
-                id: 'team.button.unread.ariaLabel',
-                defaultMessage: '{teamName} team unread',
+                id: 'team.button.mentions.ariaLabel',
+                defaultMessage: '{teamName} team, {mentionCount} mentions',
             },
             {
                 teamName: displayName,
+                mentionCount: mentions,
             });
 
-            if (mentions) {
-                ariaLabel = formatMessage({
-                    id: 'team.button.mentions.ariaLabel',
-                    defaultMessage: '{teamName} team, {mentionCount} mentions',
-                },
-                {
-                    teamName: displayName,
-                    mentionCount: mentions,
-                });
-
-                badge = (
-                    <span className={classNames('badge badge-max-number pull-right small', {urgent: this.props.hasUrgent})}>{mentions > 99 ? '99+' : mentions}</span>
-                );
-            }
+            badge = (
+                <span className={classNames('badge badge-max-number pull-right small', {urgent: props.hasUrgent})}>{mentions > 99 ? '99+' : mentions}</span>
+            );
         }
+    }
 
-        ariaLabel = ariaLabel.toLowerCase();
-
-        const content = (
-            <TeamIcon
-                className={teamClass}
-                withHover={true}
-                content={this.props.content || displayName || ''}
-                url={teamIconUrl}
-            />
-        );
-
-        let toolTip = this.props.tip || localizeMessage('team.button.name_undefined', 'This team does not have a name');
-        let orderIndicator: JSX.Element | undefined;
-        if (typeof this.props.order !== 'undefined' && this.props.order < 10) {
-            toolTip = (
-                <>
-                    {toolTip}
-                    <KeyboardShortcutSequence
-                        shortcut={KEYBOARD_SHORTCUTS.teamNavigation}
-                        values={{order}}
-                        hideDescription={true}
-                        isInsideTooltip={true}
-                    />
-                </>
+    ariaLabel = ariaLabel.toLowerCase();
```

Comment:
Out of the scope, but I think there are no good reasons to lowercase aria labels. The only reason I have read around is to make sure an all caps word is not spell out, but then if it should be spell out it won't... So I would remove this.

But 0/5

Reply:
I do want a screen reader that reads capital letters out louder 😛 

Same as above though that I'm leaving this as-is since it doesn't affect making this functional or improving the tooltips